src/embassy/mod.rs

```diff
@@ -11,7 +11,12 @@ pub mod time_driver_impl;
 pub mod time_driver_impl;
 
 // This should be called after global clocks inited
-pub fn init() {
+/// Initialize the Embassy time driver.
+///
+/// # Safety
+///
+/// This function should be called only once.
+pub unsafe fn init() {
     crate::pac::PFIC.sctlr().modify(|w| w.set_sevonpend(true));
```

Comment:
I guess we don't really need this anymore LOL, this is handled in `qingke` I believe, we set the WFItoWFE bit.